__tests__/app.test.js

```diff
@@ -42,7 +42,48 @@ describe("2. GET /api/topics", () => {
   });
 });
 
-describe("Error handling tests", () => {
+describe("3. GET /api/articles/:article_id", () => {
+  test("status: 200, responds with the correct article matching the id parameter", () => {
+    return request(app)
+      .get("/api/articles/2")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toEqual(
+          expect.objectContaining({
+            article_id: 2,
+            author: expect.any(String),
+            body: expect.any(String),
+            title: expect.any(String),
+            topic: expect.any(String),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
```

Comment:
I would change these to the exact values expected as we can find them in the data files